src/migrations/hotbar-store/5.0.0-beta.4.ts

```diff
@@ -0,0 +1,30 @@
+import { Hotbar } from "../../common/hotbar-store";
```

Comment:
This prevents a circular dependency.

```suggestion
import type { Hotbar } from "../../common/hotbar-store";
```